.changeset/small-cups-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystonejs/fields': minor
+---
+
+Updated `match` function to use `getItems` function from `server-side-graphql-client`.
```

Comment:
The `fields` package doesn't need a `major` change, as it's only update its tests. We do however need a `major` changeset on `test-utils`, as we're removing a function from its public API.

Reply:
Yeah, I was kind of thinking on the same line. But wasn't sure.